services/web/server/src/simcore_service_webserver/director/handlers.py

```diff
@@ -142,14 +148,16 @@ def _resolve_url(request: web.Request) -> URL:
 
 @login_required
 async def running_interactive_services_delete_all(request: web.Request) -> web.Response:
+    await check_permission(request, "services.interactive.*")

```

Comment:
actually wondering if this permission stuff could not be encoded in the openapi using [tags](https://swagger.io/docs/specification/grouping-operations-with-tags/) maybe, also actually found stuff about authentication with cookies [there](https://swagger.io/docs/specification/authentication/cookie-authentication/)